src/Symfony/Bridge/Doctrine/Tests/ArgumentResolver/EntityValueResolverTest.php

```diff
@@ -144,6 +144,20 @@ public function testResolveWithNullId()
         $this->assertSame([null], $resolver->resolve($request, $argument));
     }
 
+    public function testResolveWithArrayIdNullValue()
+    {
+        $manager = $this->getMockBuilder(ObjectManager::class)->getMock();
```

Comment:
```suggestion
        $manager = $this->createMock(ObjectManager::class);
```

Reply:
I fixed the other ones in
* #57729

Reply:
(Also your PR is targeting 7.2 while mine targets 6.4 :thinking: )